docs/topics/proc_configuring-jul-logging-vertx.adoc

```diff
@@ -0,0 +1,19 @@
+
+[#_configure_jul_logging_vertx]
+= Configuring logging for your {VertX} application using `java.util.logging`
+To configure debug logging for your {VertX} application using `java.util.logging`:
+
+// is it just `properties` file , more accurate designation
+* Set the `java.util.logging.config.file` system property in the `application.properties` file.
+The value of this variable must correspond to the name of your link:https://docs.oracle.com/javase/8/docs/technotes/guides/logging/overview.html#a1.8[`java.util.logging` configuration file^].
+This ensures that `LogManager` initializes `java.util.logging` at application startup.
+// the `java.util.logging` file is in a standard java.utils.Properties file.
+//Alternative:  add a logger class to your application. see docs.
+
+* Alternatively, add a `java.util.logging` configuration file with the name `vertx-default-java.util.logging-logging.properties` to the classpath of your Maven project
+//(e.g. inside your fatjar).
+{VertX} will use that file to configure `java.util.logging` on application startup.
+
+{VertX} allows you to specify a custom logging backend using the `LogDelegateFactory` that provides pre-built implementations for the `Log4J`, `Log4J2` and `SLF4J` libraries.
+Unlike `java.util.logging` which is included by default, the other backends require that you specify their respective libraries as dependencies in for your application.
```

Comment:
"dependencies in for your application" -- one of "in" or "for" must go

Reply:
Fixed:)

Reply:
fixed:)